contrib/syncer/README.md

```diff
@@ -0,0 +1,31 @@
+# KCP: Install Syncer component on the P-cluster
+
+
+1. Create workspace on your Kcp cluster (remember to use your Kcp credentials)
+
+```sh
+cat <<EOF | kubectl --kubeconfig .kcp/admin.kubeconfig apply -f -
+apiVersion: tenancy.kcp.dev/v1beta1
+kind: Workspace
+metadata:
+  name: my-workspace
+spec:
+  type: Universal
+EOF
```

Comment:
Let's make our kcp kubectl plugin a prerequisite. Then tell users to do `kubectl kcp workspace create <name> --use`